repositories/folders_repo.go

```diff
@@ -32,6 +33,27 @@ func (r *FoldersRepo) Get(ctx context.Context, id string) (*models.Folder, error
 	return rowToFolder(row), nil
 }
 
+func (r *FoldersRepo) GetBySpace(ctx context.Context, space *models.Space, q string) ([]*models.Folder, error) {
+	query := r.client.Folder.Query().
+		Where(folder.SpaceIDEQ(space.ID))
+
+	if q != "" {
+		query = query.Where(func(s *sql.Selector) {
+			s.Where(sql.Like(folder.FieldName, "%"+q+"%"))
```

Comment:
Will need a custom predicate here using the Postgres `~*` operator for case insensitive filtering.

Reply:
I know, still draft to get a working (testable) version and then refactor it. Will add a test that makes sure SQL injection fails.